Makefile.overrides.mk

```diff
@@ -20,3 +20,11 @@ PHONYS := $(shell ls | grep -v Makefile)
 .PHONY: $(PHONYS)
 $(PHONYS):
 	@$(MAKE) $@
+
+ifeq ($(BUILD_WITH_CONTAINER),0)
```

Comment:
Why are we overriding this? if I explicitly set `BUILD_WITH_CONTAINER=0` I really don't want to build using a container... especially in CI

Reply:
This is transitional. I want folks to be able to do "make xxx" and have it work right off the bat without having to install 10 different tools. Basically, this is how I plan to provide a smooth transition to having BUILD_CONTAINER=1 be the default. I'll switch over individual rules in this way.

I really don't expect people to build outside the container in general. So I'm ok for the transition period to force certain activities to go through the container.

Reply:
I don't think this will work well in CI

Reply:
Luckily, CI doesn't need these targets :-)

Reply:
"I'll switch over individual rules in this way" suggests this approach will break at some point though

Reply:
My two cents on this would be in agreement with John. If people really don't want to install tools, they can prefix the make with `BUILD_WITH_CONTAINER=1` for the short term. What happens if there is someone who can't install Docker on their build machine but could install the other tools (I'm not in that boot, but for the sake of argument)? There is no way for them to build at all in the interim without hacking a Makefile. Once the build container is enabled in the override, they can set the value to 0.

Reply:
All right, I've taken this out.